target/linux/mvebu/image/cortex-a9.mk

```diff
@@ -68,60 +91,67 @@ endef
 TARGET_DEVICES += linksys_wrt1900ac
 
 define Device/plathome_openblocks-ax3-4
+  DEVICE_VENDOR := Plat'Home
+  DEVICE_MODEL := OpenBlocks AX3
   DEVICE_DTS := armada-xp-openblocks-ax3-4
   SUPPORTED_DEVICES += openblocks-ax3-4
   BLOCKSIZE := 128k
   PAGESIZE := 1
   IMAGES += factory.img
   IMAGE/factory.img := append-kernel | pad-to $$(BLOCKSIZE) | append-ubi
-  DEVICE_TITLE := Plat'Home OpenBlocks AX3
 endef
 TARGET_DEVICES += plathome_openblocks-ax3-4
 
 define Device/marvell_a385-db-ap
   $(Device/NAND-256K)
+  DEVICE_VENDOR := Marvell
+  DEVICE_MODEL := Armada 385 Development Board (DB-88F6820-AP)
```

Comment:
Unless you have removed it by intention, I would add the "AP":
Armada 385 Development Board AP (DB-88F6820-AP)